.travis.yml

```diff
@@ -1,46 +1,41 @@
-#Trying to go to the faster container infrastructure of github
+dist: trusty
 sudo: false
-
 language: php
 
 php:
-  - 5.6
-  - 7.0
   - 7.1
-  - nigthly
```

Comment:
I added it back, but I had a different error before. Let's see if it happens again.